lib/ui/painting.dart

```diff
@@ -5178,6 +5178,9 @@ class Shadow {
 }
 
 /// A handle to a read-only byte buffer that is managed by the engine.
+///
+/// The create of this object is responsible for calling [dispose] when it is
+/// no longer needed. However,
```

Comment:
is something missing after "However"?

Reply:
I decided to put this line on the [dispose] method. Deleted.